pkg/webui/components/header/index.js

```diff
@@ -112,21 +84,21 @@ Header.propTypes = {
   dropdownItems: PropTypes.arrayOf(PropTypes.shape({
     title: PropTypes.message.isRequired,
     icon: PropTypes.string,
-    path: PropTypes.string.isRequired,
+    path: PropTypes.string,
```

Comment:
it is `required` in `<NavigationBar/>` and the app crashes if you dont pass an entry without the `path` entry

Reply:
These are the proptypes for the dropdown items, they aren't even passed to the `<NavigationBar>`.

Reply:
In general, we can improve our recurring proptypes by referencing them rather than redefining. Also this particular one should be solved rather like this:
```js
PropTypes.arrayOf(
    PropTypes.oneOfType([
      PropTypes.shape({
        title: PropTypes.message.isRequired,
        icon: PropTypes.string,
        path: PropTypes.string,
        action: PropTypes.func.isRequired,
      }),
      PropTypes.shape({
        title: PropTypes.message.isRequired,
        icon: PropTypes.string,
        path: PropTypes.string.isRequired,
        action: PropTypes.func,
      }),
    ])
  ).isRequired,
```

However, fixing this here would be too out of scope for this PR. I'll create an issue.